### PR TITLE
Enable service maintenance page

### PR DESCRIPTION
**Pull request by RMcVelia**
## Context
Add service maintenance page as per https://github.com/DFE-Digital/teacher-services-cloud/blob/main/documentation/maintenance-page.md 

## Changes proposed in this pull request
- add required config for the maintenance page deployment
- update Makefile
- add github workflows
- added temp hosts to CLAIMS_HOSTS and PLACEMENTS_HOSTS

## Guidance to review
Will be tested in staging env 
Already added and tested on several existing services

## Link to Trello card
https://trello.com/c/b2Xg8vl1/1979-ittms-maintenance-page

### CODE REVIEW

**Comment by RMcVelia** [comment]
[ITTMS maintenance page](https://trello.com/c/b2Xg8vl1/1979-ittms-maintenance-page)

File: maintenance_page/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,4 @@
+FROM nginx:1.25.4
```
Comment (saliceti) [inline maintenance_page/Dockerfile:1]: Could we try the unprivileged image as in [the tech guidance](https://github.com/DFE-Digital/technical-guidance/blob/master/Dockerfile#L1) ?
If it works, we can update the template

File: maintenance_page/scripts/failover.sh [hunk 2]
```diff
@@ -0,0 +1,32 @@
+#!/usr/bin/env bash
+
+set -eu
+
+NAMESPACE=$(jq -r '.namespace' terraform/aks/workspace_variables/${CONFIG}.tfvars.json)
```
Comment (saliceti) [inline maintenance_page/scripts/failover.sh:5]: Should it be terraform/application/config/${CONFIG}.tfvars.json

File: maintenance_page/scripts/failback.sh [hunk 3]
```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env bash
+
+set -eu
+
+NAMESPACE=$(jq -r '.namespace' terraform/aks/workspace_variables/${CONFIG}.tfvars.json)
```
Comment (saliceti) [inline maintenance_page/scripts/failback.sh:5]: Should it be terraform/application/config/${CONFIG}.tfvars.json

**Comment by saliceti** [comment]
Tested but it didn't enable the maintenance page. Probably because it changed the `itt-mentor-services-staging.test.teacherservices.cloud` ingress. But it should be `manage-school-placements-staging.test.teacherservices.cloud` and `track-and-pay-staging.test.teacherservices.cloud`

File: maintenance_page/scripts/failover.sh
Comment (saliceti) [inline maintenance_page/scripts/failover.sh:1]:
```
./maintenance_page/scripts/failback.sh
make: ./maintenance_page/scripts/failback.sh: Permission denied
```
Is it missing the execute permission?

File: maintenance_page/scripts/failback.sh
Comment (saliceti) [inline maintenance_page/scripts/failback.sh:1]:
```
./maintenance_page/scripts/failback.sh
make: ./maintenance_page/scripts/failback.sh: Permission denied
```
Is it missing the execute permission?

File: maintenance_page/scripts/failover.sh (hunk 2 above)
Comment (RMcVelia) [reply maintenance_page/scripts/failover.sh]: updated, and will fix in template

File: maintenance_page/scripts/failback.sh (hunk 3 above)
Comment (RMcVelia) [reply maintenance_page/scripts/failback.sh]: updated, and will fix in template

File: maintenance_page/scripts/failback.sh
Comment (RMcVelia) [reply maintenance_page/scripts/failback.sh:1]: updated the ingress, and fixed the execute permission (and will fix in template)

File: maintenance_page/Dockerfile (hunk 1 above)
Comment (RMcVelia) [reply maintenance_page/Dockerfile]: added the image

**Review by saliceti — APPROVED** [review]

**Pull request merged**